common/events.hpp

```diff
@@ -76,6 +78,13 @@ class Timer : public EventLoopObject {
 public:
 	Timer(EventLoop &loop);
 	~Timer() {
+		// Note: Testing pointer value here, not bool.
+		if (!destroying_) {
```

Comment:
Not fully following, as opposed to comparing to `nullptr`?

Reply:
No, as opposed to `if (!*destroying_)`, IOW comparing the bool value. But using `nullptr` is much clearer, I'll switch to that!